packages/react-aria-components/docs/TagGroup.mdx

```diff
@@ -443,6 +458,26 @@ The `description` slot can be used to associate additional help text with a `Tag
 </TagGroup>
 ```
 
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show CSS</summary>
+
+```css
+@layer taggroup {
+  .react-aria-TagGroup {
+    [slot=description] {
+      font-size: 12px;
+    }
+
```

Comment:
since the error message example is below, do we want to move the css pertaining to the error message down to that example? i know they are technically right next to each other so not a big deal but just wondering

Reply:
I think i'm going to keep these two together, it's just so small and right next to each other as you pointed out